configapi/api_inventory.go

```diff
@@ -72,49 +108,44 @@ func DeleteGnb(c *gin.Context) {
 }
 
 func handlePostGnb(c *gin.Context) error {
-	var gnbName string
-	var exists bool
-	if gnbName, exists = c.Params.Get("gnb-name"); !exists {
+	gnbName, exists := c.Params.Get("gnb-name")
+	if !exists {
 		errorMessage := "post gNB request is missing gnb-name"
 		logger.ConfigLog.Errorln(errorMessage)
 		return errors.New(errorMessage)
```

Comment:
What about using a single way to "report" errors? I see you are using `fmt.Errorf` but some previous code is using `errors.New` in other places
```suggestion
		return fmt.Errorf(errorMessage)
```